lib/xandra/cluster/control_connection.ex

```diff
@@ -410,7 +412,10 @@ defmodule Xandra.Cluster.ControlConnection do
              :ok <- register_to_events(data, connected_node),
              :ok <- inet_mod(transport).setopts(socket, active: :once) do
           [local_host | _] = peers
-          {:ok, %ConnectedNode{connected_node | host: local_host}, peers}
+
+          res = {:ok, %ConnectedNode{connected_node | host: local_host}, peers}
+          Logger.debug("#{inspect(res)}")
+          res
```

Comment:
I don't think this is something we should log. If we want to, we should log it with a friendly message, but I think we can skip the logging for now and go back to the previous code. Thoughts?

Reply:
I added this section because between the last log DEBUG line and the next logging DEBUG line, there was quite a few places where state was being changed that couldn't be debugged without this. I'll add an additional message to it